dashboard/app/assets/javascripts/header.js

```diff
@@ -172,6 +172,45 @@ dashboard.buildHeader = function (stageData, progressData, currentLevelId, userI
   }
 };
 
+dashboard.initSendToPhone = function(selector) {
```

Comment:
Can we pass in the element itself here rather than a selector for the parent? (i think so)